downloads/swan-lake-release-notes/swan-lake-2201.6.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.6.3 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-6-3/
+active: 2201-6-3
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-6-3
+    - /downloads/swan-lake-release-notes/2201.6.3/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-6-3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.6.3
+
+<em>Swan Lake 2201.6.3 is the third patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,
+and includes a few security improvements</em>
```

Comment:
```suggestion
and a few security improvements</em>.
```

Reply:
Updated in [a48e8](https://github.com/ballerina-platform/ballerina-dev-website/pull/8201/commits/a48e85190b2424b2ed5fe71ddaf0eb1f814ca58c).